exec/java-exec/src/test/java/org/apache/drill/exec/store/json/TestJsonRecordReader.java

```diff
@@ -116,6 +117,7 @@ public void testMixedNumberTypes() throws Exception {
           .jsonBaselineFile("jsoninput/mixed_number_types.json")
           .build().run();
     } catch (Exception ex) {
+      ex.printStackTrace();
```

Comment:
Not a good idea to print stack trace in unit tests. The output of our unit tests is already too verbose.
Use junit.fail with the message from the exception instead?